docs/yul.rst

```diff
@@ -99,9 +99,11 @@ Grammar::
     If =
         'if' Expression Block
     Switch =
-        'switch' Expression Case* ( 'default' Block )?
+        'switch' Expression ( Case+ Default | Case+ | Default )
```

Comment:
What about `(Case+ Default? | Default)`?

Also please squash the commits.

Reply:
Yup, that's shorter. I pushed an update.